src/main/java/org/openmrs/maven/plugins/Delete.java

```diff
@@ -25,13 +25,8 @@
      */
     private String serverId;
 
-    /**
-     * @required
-     * @component
-     */
-    Wizard wizard;
-
     public void execute() throws MojoExecutionException, MojoFailureException {
+        initTask();
```

Comment:
It's easy to forget about that... Let's add public abstract void executeTask() throws MojoExecutionException, MojoFailureException; to AbstractTask and implement execute() in AbstractTask to call initTask() and executeTask(). Next all classes should implemented executeTask() instead of execute().